src/reducers/Material.js

```diff
@@ -38,6 +38,7 @@ function materialsCloneOne(state, action) {
 function materialsToggleIsNonPeriodicForOne(state, action) {
     const newMaterial = state.materials[state.index].clone();
     newMaterial.isNonPeriodic = !newMaterial.isNonPeriodic;
+    newMaterial.lattice = Made.tools.material.scaleLatticeToMakeNonPeriodic(newMaterial);
```

Comment:
Per https://github.com/Exabyte-io/made.js/pull/40/files#r764470311 this can just be:

```js
newMaterial.isNonPeriodic = !newMaterial.isNonPeriodic;
Made.tools.material.scaleLatticeToMakeNonPeriodic(newMaterial);
return materialsUpdateOne(state, Object.assign({}, state, {material: newMaterial}));
```